bwtv.py

```diff
@@ -30,20 +30,8 @@
 cached_credentials = {}
 
 
-def _fetch_secret(site, secret_id):
+def load_site_credentials(site):
     try:
-        return cache[site][secret_id]
-    except KeyError:
-        pass
-
-    session = sessions.setdefault(getpid(), Session())
-
-    try:
-        full_url = "{}/api/secrets/{}/".format(
-            config.get(site, "url"),
-            secret_id,
-        )
-
         if site not in cached_credentials:
             if (
```

Comment:
This `if` is strange, btw. `and` has higher priority than `or`, doesn’t it? I think the first line was meant to be put in parens?

Reply:
yes